features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/GraphQLConsole/GraphQLAuthentication.jsx

```diff
@@ -0,0 +1,392 @@
+            const promiseApp = Application.get(event.target.value);
+            let accessToken;
+            let keyType;
+
+            setSelectedApplication(event.target.value);
+
+            if (subscriptions != null && subscriptions.find((sub) => sub.applicationId
+                === selectedApplication).status === 'PROD_ONLY_BLOCKED') {
+                const sKType1 = 'SANDBOX';
+                setSelectedKeyType(sKType1);
+                keyType = 'SANDBOX';
+            } else {
+                keyType = selectedKeyType;
+            }
+
+            promiseApp
+                .then((application) => {
+                    return application.getKeys();
+                })
+                .then((appKeys) => {
+                    if (appKeys.get(keyType)) {
+                        ({ accessToken } = appKeys.get(keyType).token);
+                    }
+                    setAccessTocken(accessToken);
+                    SetKeys(appKeys);
+                });
+        } else {
+            let accessToken;
+            setSelectedKeyType(event.target.value);
+
+            if (keys.get(event.target.value)) {
+                ({ accessToken } = keys.get(event.target.value).token);
+            }
+            setAccessTocken(accessToken);
+        }
+    };
+
+
+    /**
+     * Set the environment and URLs for selected environment name
+     * @param {*} event
+     */
+    const handleEnvironemtChange = (event) => {
+        setSelectedEnvironment(event.target.value);
+        const urls = environmentObject.find((elm) => event.target.value === elm.environmentName).URLs;
+        setURLss(urls);
+    };
+
+
+    /**
+     * Handle onClick of shown access token
+     */
+    const handleClickShowToken = () => {
+        setShowToken(!showToken);
+    };
+
+
+    /**
+     * Load the access token for selected key type
+     * @param {*} event
+     */
+    const handleaccessTockenChanges = (event) => {
+        setAccessTocken(event.target.value);
+    };
+
+    /**
+     * Set the security scheme type for selected security scheme type
+     * @param {*} event
+     */
+    const handlesecuritySchemeType = (event) => {
+        setSecuritySchemeType(event.target.value);
+    };
+
+
+    return (
+        <>
+            <Grid container className={classes.grid}>
+                {!isPrototypedAPI && !user && (
+                    <Grid item md={6}>
+                        <Paper className={classes.userNotificationPaper}>
+                            <Typography variant='h5' component='h3'>
+                                <Icon>warning</Icon>
+                                {' '}
+                                <FormattedMessage id='notice' defaultMessage='Notice' />
+                            </Typography>
+                            <Typography component='p'>
+                                <FormattedMessage
+                                    id='api.console.require.access.token'
```

Comment:
Use proper id for FormattedMessages.

`Apis.Details.GraphQLConsole.GraphQLAuthentication.api.console.require.access.token`

Reply:
Fix in the other places as well.